.storybook/preview.js

```diff
@@ -18,6 +18,6 @@ addParameters({
   },
 })
 
-addDecorator(withSmartKnobs())
+addDecorator(withSmartKnobs({ ignoreProps: ['leftAddon'] }))
```

Comment:
I added this to prevent storybook to go infinite loop on these. LeftAddon generally correspond to other components so i think we can ignore subcomponents like that.

Reply:
Can you comment this in the code then, 'cos it is a pretty generic change